compiler/luci/pass/src/FuseGeluPass.cpp

```diff
@@ -30,9 +30,81 @@
 namespace
 {
 
+#define CHECK_OR_FALSE(condition) \
+  if (not(condition))             \
+    return false;
+
 // Float comparison
 bool same(float a, float b) { return fabs(a - b) < 1e-5; }
 
+class GeluPatternBase
```

Comment:
Please add virtual destructor for parent class.

Reply:
Done